libs/cardano-ledger-test/src/Test/Cardano/Ledger/Generic/TxGen.hs

```diff
@@ -13,7 +13,9 @@
 {-# LANGUAGE TypeFamilies #-}
 {-# LANGUAGE UndecidableInstances #-}
 
-module Test.Cardano.Ledger.Generic.TxGen
+module Test.Cardano.Ledger.Generic.TxGen where
+
+{-
```

Comment:
Explicit exports got commented out

Reply:
Yeah, that was a temporary change in order to compile it with bisection strategy to find what was taking forever. Forgot to undo it.